tests/test_other.py

```diff
@@ -9150,6 +9150,12 @@ def test_node_js_pthread_module(self):
     ret = run_process(NODE_JS + ['--experimental-wasm-threads'] + [os.path.join('subdir', moduleLoader)], stdout=PIPE).stdout
     self.assertContained('hello, world!', ret)
 
+  def test_node_js_system(self):
+    ensure_dir('subdir')
```

Comment:
the subdirectory is not needed, I think? Can just emit in the current directory (which will be a temp directory created by the test runner).

Reply:
Ah, was not sure what that did. I'll remove it.